src/core/Roles.sol

```diff
@@ -49,4 +53,7 @@ library Roles {
     /// @notice game consumer notary role.
     /// can issue signatures for in-game crafting and speed ups
     bytes32 internal constant GAME_CONSUMER_NOTARY = keccak256("GAME_CONSUMER_NOTARY_ROLE");
+
+    /// @notice DEPLOYER role.
+    bytes32 internal constant DEPLOYER = keccak256("DEPLOYER_ROLE");
```

Comment:
As per previous comment, I don't think we need this deployer role anymore?

Reply:
same as above.